src/utils/protocol-events-map.ts

```diff
@@ -18,13 +26,11 @@ export enum ProtocolEvent {
   BatchDeployment = 'BatchDeployment',
   LegacyMintNft = 'LegacyMintNft',
   MoeMintNft = 'MoeMintNft',
-  TransferOwnership = 'TransferOwnership',
-  BridgeOutErc721 = 'BridgeOutErc721',
-  // BridgeOutContract = 'BridgeOutContract',
+  BridgeOut = 'BridgeOut',
   AvailableOperatorJob = 'AvailableOperatorJob',
-  BridgeInErc721 = 'BridgeInErc721',
-  // BridgeInContract = 'BridgeInContract',
+  BridgeIn = 'BridgeIn',
   FailedOperatorJob = 'FailedOperatorJob',
+  TransferErc20OrErc721 = 'TransferErc20OrErc721',
```

Comment:
I'm wondering why these were combined too

Reply:
The event signature hash is the same for both TransferERC20 and TransferERC721, with the sole distinction lying in the transaction data field. If the data is either empty or '0x', it's a TransferERC721; otherwise, it's a TransferERC20. Consolidating these protocol events avoids the need for extra checks.